packages/store/test/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
     "extends": "../tsconfig.json",
     "compilerOptions": {
-        "strict": false
+        "strict": false,
+        "experimentalDecorators": true
```

Comment:
Don't think we need this, the `tsconfig.json` at the project root already sets this property to `true`.

Reply:
Removed in dbac36fcf5c5e9752be2f18a8961439665680bdc